cmd/db/db.go

```diff
@@ -0,0 +1,246 @@
+package db
+
+import (
+	"context"
+	"fmt"
+
+	migrate "github.com/rubenv/sql-migrate"
+	"github.com/spf13/cobra"
+	"github.com/stellar/go/support/config"
+	"github.com/stellar/go/support/log"
+
+	"github.com/stellar/stellar-disbursement-platform-backend/cmd/utils"
+	"github.com/stellar/stellar-disbursement-platform-backend/db"
+	adminmigrations "github.com/stellar/stellar-disbursement-platform-backend/db/migrations/admin-migrations"
+	authmigrations "github.com/stellar/stellar-disbursement-platform-backend/db/migrations/auth-migrations"
+	sdpmigrations "github.com/stellar/stellar-disbursement-platform-backend/db/migrations/sdp-migrations"
+	"github.com/stellar/stellar-disbursement-platform-backend/internal/services"
+	sdpUtils "github.com/stellar/stellar-disbursement-platform-backend/internal/utils"
+)
+
+const DBConfigOptionFlagName = "database-url"
+
+type DatabaseCommand struct{}
+
+func (c *DatabaseCommand) Command(globalOptions *utils.GlobalOptionsType) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:              "db",
+		Short:            "Database related commands",
+		PersistentPreRun: utils.PropagatePersistentPreRun,
+		RunE:             utils.CallHelpCommand,
+	}
+
+	// ADD SUBCOMMANDs:
+	// The following commands use --all and --tenant-id flags.
+	cmd.AddCommand(c.setupForNetworkCmd(cmd.Context(), globalOptions))         // 'setup-for-network'
+	cmd.AddCommand(c.sdpPerTenantMigrationsCmd(cmd.Context(), globalOptions))  // 'sdp migrate up|down'
+	cmd.AddCommand(c.authPerTenantMigrationsCmd(cmd.Context(), globalOptions)) // 'auth migrate up|down'
+
+	// The following command does NOT use --all and --tenant-id flags.
+	cmd.AddCommand(c.adminMigrationsCmd(cmd.Context(), globalOptions)) // 'admin migrate up|down'
+	cmd.AddCommand(c.tssMigrationsCmd(cmd.Context(), globalOptions))   // 'tss migrate up|down'
```

Comment:
Both commands don't take the flags or only the `admin`? Is the `tss` per tenant?

Reply:
Neither TSS nor admin are per tenant. They're both global.